doom-themes-base.el

```diff
@@ -41,6 +41,7 @@
     (font-lock-string-face               :foreground strings)
     (font-lock-type-face                 :foreground type)
     (font-lock-variable-name-face        :foreground variables)
+    (font-lock-number-face               :foreground numbers)
```

Comment:
Judging by the previous `:foreground` lines, it should be sorted in lexicographic order.